### DISCUSSION TITLE
Installing tea in my linux(ubuntu)

### DISCUSSION THREAD

**Discussion by karaz-debug** [pkgxdev/pkgx#267] [Help]
Actually, I tried to install tea on my device with this command :
             
         sh <(curl https://tea.xyz)
but  when I run it, it says 
             "the latest version of tea was already installed"
but when I go to my home root I can't see the files there.

so is it my device problem or the way I am installing tea is wrong

Thanks any help and guide lines I will appritiate

**Comment by mxcl** [comment]
Is `~/.tea` not there? What does `which tea` and `tea --version` say?

**Reply by karaz-debug** [reply]
Yeah actually it's not there

**Reply by karaz-debug** [reply]
the tea --verson say's  tea 0.14.1
and tea /home/akdev/.tea/tea.xyz/v0.14.1/bin/tea

**Reply by mxcl** [reply]
is `~/.tea` not the same as `/home/akdev/.tea`?

**Reply by karaz-debug** [reply]
i think is not how can i check out

**Reply by mxcl** [reply]
You said:

> when I go to my home root I can't see the files there

What makes you say this? What did you do?

**Reply by karaz-debug** [reply]
just i installed but I am confused when you install tea, are you supposed to see the files because, like when using window I install thing the I usually used to see but this one I am confused with it, that's why I asked
